### PR TITLE
Introduce new Web3 authentication section and Coinbase Wallet page

### PR DESCRIPTION
**Pull request by chanioxaris**
> [!IMPORTANT]
> 🔎 Previews:
>
> -

### This PR:

-

### CODE REVIEW

File: docs/manifest.json [hunk 1]
```diff
@@ -519,6 +519,15 @@
...
                     ]
+                  },
+                  {
+                    "title": "Web3",
+                    "items": [
+                      {
+                        "title": "Coinbase Wallet",
+                        "href": "/docs/authentication/web3/coinbase-wallet"
+                      }
+                    ]
```
Comment (bradlc) [inline docs/manifest.json:530]: An additional array is required here to conform to the [JSON schema](https://github.com/clerk/clerk-docs/blob/8c304af5180ac25f29a880167eca36bb71f618fc/docs/manifest.schema.json):

```suggestion
                    "items": [
                      [
                        {
                          "title": "Coinbase Wallet",
                          "href": "/docs/authentication/web3/coinbase-wallet"
                        }
                      ]
                    ]
```

Comment (chanioxaris) [reply docs/manifest.json]: Thanks!

**Comment by alexisintech** [comment]
Slack discussion [here](https://clerkinc.slack.com/archives/C01QFRQNHSN/p1727453318546949)

**Review by astrotars — APPROVED** [review]
🚢

**Pull request merged**